src/pages/CreateBenefitForm.tsx

```
@@ -0,0 +1,9 @@
+import FormSchema from "../components/form/Schema/FormSchema";
+import Layout from "../components/layout/Layout";
+export default function CreateBenefitForm() {
+  return (
+    <Layout showMenu={true} showSearchBar={true} showLanguage={false}>
+      <FormSchema />;
+    </Layout>
```

Biome

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)